src/main/java/org/altbeacon/beacon/BeaconManager.java

```diff
@@ -138,27 +126,184 @@ public static void setDebug(boolean debug) {
     /**
      * The default duration in milliseconds spent not scanning between each bluetooth scan cycle when no ranging/monitoring clients are in the foreground
      */
-    public static final long DEFAULT_BACKGROUND_BETWEEN_SCAN_PERIOD = 5*60*1000;
+    public static final long DEFAULT_BACKGROUND_BETWEEN_SCAN_PERIOD = 5 * 60 * 1000;
 
     private long foregroundScanPeriod = DEFAULT_FOREGROUND_SCAN_PERIOD;
     private long foregroundBetweenScanPeriod = DEFAULT_FOREGROUND_BETWEEN_SCAN_PERIOD;
     private long backgroundScanPeriod = DEFAULT_BACKGROUND_SCAN_PERIOD;
     private long backgroundBetweenScanPeriod = DEFAULT_BACKGROUND_BETWEEN_SCAN_PERIOD;
 
+    private static Logger sLogger = Loggers.empty();
+
+    /**
+     * Set the logger to be used with in the AltBeacon library. This will receive all log message
+     * the library outputs.
+     *
+     * @param logger The logger to be used.,
+     * @throws NullPointerException if logger is null.
+     * @see org.altbeacon.beacon.Loggers
+     * @since 2.1
+     */
+    public static void setsLogger(Logger logger) {
```

Comment:
Yeah, I think it was a blind refactor. I'll update this. I realized too, that this is a breaking change, but since this is a whole minor change update I'm guessing it's okay.